src/AWS.Deploy.Orchestration/Data/AWSResourceQueryer.cs

```diff
@@ -512,7 +512,7 @@ public async Task<Vpc> GetDefaultVpc()
                             }
                         }
                     })
-                .Vpcs.FirstOrDefaultAsync(),
+                .Vpcs.FirstAsync(),
```

Comment:
What is the reason for this change? Won't this cause an error message to the user where we just add "Sequence contains no elements" to the end of there are is no default VPC?

Reply:
Building the solution with .NET8 threw a "possible null reference" exception for that block of code. I switched back to `FirstOrDefaultAsync` and instead I am not returning a nullable VPC.